.gitignore

```diff
@@ -59,7 +59,6 @@ errorScreenshots
 generated-themes
 package-lock.json
 lib
-!/packages/webpack-config-terra/lib/
```

Comment:
removed this because i updated the package to do the no compile copy from src to lib